### PR TITLE
Update from Docusaurus v1 to v2

### LINKED ISSUE #88: Add a button to make it easier for people to contribute to the docs
It would be great if every page on https://docs.radicle.xyz/ had some kind of button in one of the page corners which would make it obvious how to contribute to the docs (e.g. by fixing a typo or reporting an issue).  Some other projects have a ["Fork me on Github" ribbon](https://forkmeon.github.io/) or similar so this might be one approach.

### PR DESCRIPTION
**Pull request by joelhans**
I spent some time working through the manual upgrade process from v1->v2 of Docusaurus, which enables a lot of features and plugins that might be useful in the future.

Not sure how much urgency there is to push this, and it would probably be best to have >1 other person checkout this PR and run it locally to make sure things generally function as expected.

Fixes #88

### CODE REVIEW

**Review by rudolfs — COMMENTED** [review]
Nice work!!

The index page seems to be missing, or maybe the redirect isn't working in dev mode?
Screenshot 2022-04-11 at 09 44 45

And the styling would have to be adjusted to fit better into the radicle ecosystem:
Screenshot 2022-04-11 at 09 45 24

Maybe @juliendonck or @brandonhaslegs can help with that?

**Comment by rudolfs** [comment]
Screenshot 2022-04-11 at 10 16 59

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
-website/build/
-website/i18n/*
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
[the remark this answers is not in the document]
Comment (joelhans) [reply .gitignore:20]: Done. Didn't know that was an available option in VSCode!

**Comment by joelhans** [comment]
> The index page seems to be missing, or maybe the redirect isn't working in dev mode? Screenshot 2022-04-11 at 09 44 45

I changed the `baseUrl` so that it always loads up at the right place.


> And the styling would have to be adjusted to fit better into the radicle ecosystem: Screenshot 2022-04-11 at 09 45 24
>
> Maybe @juliendonck or @brandonhaslegs can help with that?

I'd love that! I tried to replicate the current styling, but it could definitely align better throughout.

**Comment by rudolfs** [comment]
Getting a bunch of warnings and errors when I try to build the index:

```
Exhaustive list of all broken links found:

- On source page path = /docs/understanding-radicle/faq:
   -> linking to using-radicle/tracking-and-viewing.md (resolved as: /docs/understanding-radicle/using-radicle/tracking-and-viewing.md)

[WARNING] Docs markdown link couldn't be resolved: (using-radicle/tracking-and-viewing.md) in /Users/rudolfs/work/radicle-docs/docs/understanding-radicle/faq.md for version current
[WARNING] Docs markdown link couldn't be resolved: (using-radicle/creating-projects.md) in /Users/rudolfs/work/radicle-docs/docs/understanding-radicle/faq.md for version current
[WARNING] Docs markdown link couldn't be resolved: (using-radicle/overview.md) in /Users/rudolfs/work/radicle-docs/docs/understanding-radicle/faq.md for version current
```

**Comment by rudolfs** [comment]
Another thing I noticed is that when I start the server locally with `yarn start` the search bar doesn't work. However running with `yarn serve` the local search index work.

File: sidebars.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    {
+      type: 'category',
+      label: 'Get involved',
+      collapsed: false,
+      items: ['get-involved/join-the-community', 'get-involved/radicle-garden']
+    }
+  ],
+};
+
+module.exports = sidebars;
```
Comment (rudolfs) [inline sidebars.js:28]: Newline missing.

**Comment by rudolfs** [comment]
>> Maybe @juliendonck or @brandonhaslegs can help with that?

> I'd love that! I tried to replicate the current styling, but it could definitely align better throughout.

We can also do it in a follow-up. This looks good enough to deploy.

**Review by rudolfs — APPROVED** [review]
👍

**Comment by joelhans** [comment]
@rudolfs Thanks for the help with this. I'll try deploying once I have a decent chunk of time in my schedule, just in case anything goes terribly wrong with the process. Hopefully before the end of the week!

**Pull request merged**